build/constants.yaml

```diff
@@ -167,15 +167,18 @@
     file-name-prefix: __
 - name: php-versions
   constants:
-    php-runtime-base-tag: 20221202.1
-    php-fpm-runtime-base-tag: 20221202.1
+    php-runtime-base-tag: 20221209.2
+    php-fpm-runtime-base-tag: 20221209.2
     composer-version: 2.0.8
     composer-setup-sha384: 55ce33d7678c5a611085589f1f3ddf8b3c52d662cd01d4ba75c0ee0459970c2200a51f492d557530c71c15d8dba01eae
     composer1-9-version: 1.9.3
     composer1-10-version: 1.10.19
     composer2-0-version: 2.0.8
     composer2-2-version: 2.2.9
     composer2-3-version: 2.3.4
+    php82-version: 8.2.0
+    php82-keys: 1198C0117593497A5EC5C199286AF1F9897469DC 39B641343D8C104B2B146DC3F9C39DC0B9698544
```

Comment:
Could you please confirm: is it normal that the second php82-keys value is the same as the second php81-keys value?

Reply:
From here - https://www.php.net/downloads.php#gpg-8.2, it looks like the second values are the same for php81 and php82.

Reply:
Thanks for confirming!